hedera-node/docs/design/app/workflows.md

```diff
@@ -18,22 +18,28 @@ The ingest-workflow is single-threaded, but multiple calls can run in parallel.
 The ingest workflow consists of the following steps:
 
 0. **Check node.** The node is checked to ensure it is not in a state that prevents it from processing transactions.
-1. **Parse transaction.** The transaction arrives as a byte-array. The required parts are parsed and the structure and syntax are validated.
-2. **Deduplicate.** The transaction is checked to ensure it has not been processed before.
-3. **Check throttles.** Throttling must be observed and checked as early as possible.
-4. **Get payer account.** The account data of the payer is read from the latest immutable state.
-5. Account Balance
+1. **Check timeBox** Checks whether the transaction duration is valid as per the configuration for valid durations 
+for the network, and whether the current node wall-clock time falls between the transaction start and the transaction end (transaction start + duration)
+2. **Parse transaction.** The transaction arrives as a byte-array. The required parts are parsed and the structure and syntax are validated.
+3. **Deduplicate.** The transaction is checked to ensure it has not been processed before.
+4. **Check throttles** Throttling must be observed and checked as early as possible.
+    1. **Pure Checks** Calls `pureChecks` method that does validations based on the respective handler's transaction body 
```

Comment:
I don't think this is the "nesting" you intended.

Reply:
Thanks @david-bakin-sl . Fixed it.